src/index.ts

```diff
@@ -35,18 +43,34 @@ const parse = schedule.scheduleJob(triggerSchedule, async () => {
     torTriggerer.close();
   }
   await https.get('https://hchk.io/c12a23b6-276d-4269-9316-d3353af47052');
+  await calculateAllLiquidity();
 });
 
-mongoose.connect(db, { useMongoClient: true });
-mongoose.set('debug', true);
+  cluster.on('online', (worker) => {
+    global.console.log(`Worker   ${worker.process.pid}  is online`);
```

Comment:
Wait, why are we doing this? Why global? Why are we using console at all. We should switch to winston.

Reply:
Fixed.Now used winston 2.4 for logging.It seems like this =)
![logger](https://user-images.githubusercontent.com/14820167/33843972-f896cbb4-deaf-11e7-90f4-3c6de1646afb.png)